### DISCUSSION TITLE
Do I need to disable Sodium culling settings?

### DISCUSSION THREAD

**Discussion by TheCrossCarrier** [FxMorin/MoreCulling#247] [Q&A] [answered]
Sodium has settings for culling entities and block faces. Do I need to disable any of these when using with your mod?

**Comment by TheCrossCarrier** [comment]
And I have a small sub-question: why are Mangrove Roots Culling and End Gateway Culling disabled by default?

**Comment by TheCrossCarrier** [comment]
hello?

**Comment by FxMorin** [comment, marked as answer]
No you don't. Its recommended that you leave the sodium options enabled tho
